### DISCUSSION TITLE
Why does the new version cancel the grouping function?

### DISCUSSION THREAD

**Discussion by Nger178** [dialogic-godot/dialogic#2366] [General]
![classify](https://github.com/user-attachments/assets/1e562ad7-33eb-4c4f-81e8-925a2f1dfde6)I think these documents will become messy if they are not classified.   Is it just because I didn't find it?

**Comment by Jowan-Spooner** [comment]
@Nger178 We decided to move away from the fake folders in the sidebar because it was a hassle to have another layer of abstraction on top of the existing file system. In contrast to dialogic 1 you can now freely put timelines in the filesystem, so we expect people to organize their resources like that. 

However in the near future the sidebar will get UI/UX improvements allowing to have resources sorted by the folder they are in or by type. [This is the relevant PR](https://github.com/dialogic-godot/dialogic/pull/2350)

**Reply by Nger178** [reply]
I see, thank you.
